.travis.yml

```diff
@@ -1,20 +0,0 @@
-sudo: false
-language: node_js
-cache:
-  directories:
-    - node_modules
-    - ~/.npm
-notifications:
-  email: false
-node_js:
-  - 'node'
-before_script:
-  - npm prune
-script:
-  - commitlint-travis
```

Comment:
Since this repo uses `semantic-release`, I'd recommend keeping `commitlint` in the CI steps. There's no `commitlint-travis` on CircleCI, of course, but there is an Orb for it. You can find an example here:

https://github.com/LLK/scratch-gui/blob/develop/.circleci/config.yml#L4
https://github.com/LLK/scratch-gui/blob/develop/.circleci/config.yml#L216-L219

Once you add that, `commitlint` will start complaining about the commits in this PR. You'll need to rephrase them according to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec.

The easiest way is probably to run `git rebase -i 0d9ea0b^` and then pick `r` for "reword" on all your commits.